### ISSUE TITLE
Not building on Ubuntu 14.04

### ISSUE THREAD

**Issue by redfacedquark** [OpenBazaar/OpenBazaar-Installer#16]
``` bash
$ sh make_openbazaar.sh win32
Already up-to-date.
Switched to /home/agrajag/Personal/Code/OpenBazaar-Installer
grunt is not installed.
Cloning OpenBazaar-Server
Cloning into 'OpenBazaar-Server'...
fatal: Remote branch noupnp not found in upstream origin
Unexpected end of command stream
OpenBazaar encountered an error and could not be cloned
```

If I change branch to 'master' it builds. The message about grunt might be because I'm using v4 rather than v5. Also noticed that the nsis apt dependency was missing from the docs.

It built an installer that worked. It took a little while, possibly because python wasn't installed. I say 'worked', I think I hit #12, will have to investigate.

**Comment by ABISprotocol** [comment]
@redfacedquark yep, there are some issues there, definitely need to be addressed, see also [this](https://github.com/OpenBazaar/OpenBazaar-Server/issues/100#issuecomment-178503182). cc: @hoffmabc
